packages/lib/src/interactions/SelectToZoom.tsx

```diff
@@ -9,14 +9,13 @@ import { useMoveCameraTo } from './hooks';
 import type { CommonInteractionProps, Selection } from './models';
 import { getEnclosedRectangle, getRatioRespectingRectangle } from './utils';
 
-interface Props extends CommonInteractionProps {
-  keepRatio?: boolean;
-}
+type Props = CommonInteractionProps;
```

Comment:
Is the `Props` type still needed ?

Reply:
Just seems like a good thing to keep in case we need to add back more props in the future, and also to have one exported component with props = one matching exported props type.